packages/mui-material/src/Stack/Stack.test.js

```diff
@@ -221,6 +221,45 @@ describe('<Stack />', () => {
         display: 'flex',
       });
     });
+
+    it('should place correct margin direction even though breakpoints are not fully provided', () => {
+      expect(
+        style({
+          ownerState: {
+            direction: { lg: 'row' },
+            spacing: { xs: 0, md: 2, xl: 4 },
+          },
+          theme,
+        }),
+      ).to.deep.equal({
+        [`@media (min-width:${defaultTheme.breakpoints.values.xs}px)`]: {
+          '& > :not(style) + :not(style)': {
+            margin: 0,
+            marginTop: '0px',
+          },
+        },
+        [`@media (min-width:${defaultTheme.breakpoints.values.md}px)`]: {
```

Comment:
We are missing the style for the breakpoint `sm`.

Reply:
With current implementation, CSS styles get created only for breakpoints passed to `spacing` or `direction`. So, in this case, since we use the following props, 
```jsx
direction: { lg: 'row' },
spacing: { xs: 0, md: 2, xl: 4 },
```

we only have styles for `xs`, `md`, `lg` and `xl`. For resolution `sm`, the value of the breakpoint right below (`xs` in this case) will be used. If we want to change it so that we have every breakpoint in CSS regardless of the props, it's a bigger change and hence outside the scope of this PR.

Reply:
No, this makes sense, I was wrong in my assumption.